### ISSUE TITLE
ConnectionAbortedError: Windows

### ISSUE THREAD

**Issue by jc508** [rawpython/remi#494]
Hi,
I am running Win10 Pro with anaconda.
Installed Remi is 2022.3.7
It is crashing on any close window. Reproducable with widgets_overview_app.py
I have tested with Edge and Chrome.
```
remi.server.ws   ERROR    Error managing incoming websocket message
Traceback (most recent call last):
  File "C:\SWW\anaconda3\lib\site-packages\remi\server.py", line 167, in read_next_message
    head = self.rfile.read(2)
  File "C:\SWW\anaconda3\lib\socket.py", line 704, in readinto
    return self._sock.recv_into(b)
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
remi.server.ws   DEBUG    ws ending websocket service

```
Any clues ?
JC

**Comment by dddomodossola** [comment]
Hello @jc508 ,

That is a physiological traceback of the died websocket connection that naturally occurs when you close a browser tab. It is not a crash. If you open again the webpage at the same address you will see your application still running.

Kind Regards

**Comment by jc508** [comment]
Thanks @dddomodossola Sorry bad wording to mention 'crash' then
I used to be able to close the app by detecting the close window. (Using code you have previously published here or elsewhere.)
Now it wont close and I thought that was probably because of those errors which caused the signal/event to never get back to my code.
You are correct the app is still running - forever now.
I have got around the issue by adding a specific 'End Program' button which is probable cleaner anyway.
JC
